src/shared/components/scroll-to-top/scroll-to-top.component.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+
+export const ScrollToTop: React.FC = () => {
+    const pathname = useLocation();
+
+    useEffect(() => {
+        window.scrollTo(0, 0);
+    }, [pathname]);
+    
+    return null;
+}
```

Comment:
nit: Add empty line to the end of the file

Reply:
Good point, i will fix it